### ISSUE TITLE
Surface Models not being detected again

### ISSUE THREAD

**Issue by itmmdj** [SCConfigMgr/ConfigMgr#139]
Using version 2.2.3 and tried 2.1.7

Pertinent Portions of the Logs
_________________________________________________________
Manufacturer determined as: Microsoft	
Computer model determined as: Surface Book 2	
Unable to determine system SKU value	
Attempting to detect OS Image data from task sequence, running in DeploymentType: BareMetal
Translated OSName from OSImageVersion: Windows 10
Translated OSVersion from OSImageVersion: 1803
Translated OSImageArchitecture: x64
Using computer detection method: ComputerModel
________________________________________________________
After that it goes through each package on SCCM and doesn't match.

________________________________________________________
Attempting to find a match for driver package: Drivers - Microsoft Surface Book - Windows 10 1803 x64 (ITM00099)
Driver package does not meet computer model criteria: Drivers - Microsoft Surface Book - Windows 10 1803 x64 (ITM00099)
________________________________________________________

The name of the package that Driver Automation Tool creates is called:
"Drivers - Microsoft Surface Book 2 13 - Windows 10 1803 x64"

I have looked through the code and am not smart enough to figure out why it won't match.

**Comment by itmmdj** [comment]
So i did a some more digging and found that the if statement that determines whether or not which is on line 737 of Invoke-CMApplyDriverPackage (version 2.2.3) will result in false.

The variables at runtime are:

$PackageNameComputerModel - Surface Book 2 13
$ComputerModel - Surface Book 2

This results in a false statement due to using the "-like" operator. The issue is that I can't change it to "-match" because that would match more than one model.

Win32_ComputerSystem\SystemSKUNumber has a value of "Win32_ComputerSystem" in wmi. I am a lousy coder, so I really don't know what to do. It would seem that one would have to make a bunch of entries for each surface model taking from "SystemSKUNumber" and then constructing a string that matches the package names that Driver Automation Tool already creates.

For now I am going to change the package name to 

"Drivers - Microsoft Surface Book 2 - Windows 10 1803 x64"

**Comment by itmmdj** [comment]
It might be easier to read the comments on the package and have the user list the different model numbers there.

**Comment by NickolajA** [comment]
Yeah I assumed this would happen with the change in the comparison operator switch from "-match" to "-like" when matching those two variables. However, I'd much rather have DAT setting the right model names on the package, instead of dealing with that in the code. I'll speak to Maurice about this, as the comparison operator switch is required for some vendors because of their naming standards.

**Comment by maurice-daly** [comment]
I've just sent my contact in Microsoft a mail to see if I can get a definitive list of WMI model names. I will then update the XML and get this fixed in the DAT.
